pyproject.toml

```diff
@@ -11,6 +11,8 @@ typer = "^0.3.2"
 pandas = "^1.2.1"
 requests = "^2.25.1"
 sphinx-rtd-theme = "^0.5.1"
+PyGithub = "^1.55"
```

Comment:
I don't think you actually are using this library anywhere? If not, please remove as a dependency.